### ISSUE TITLE
I want to bypass postPersist() from adding entry in the audit entities

### ISSUE THREAD

**Issue by ravindrakhokharia** [simplethings/EntityAudit#188]
Hi, 

I wanted to know if there is a possibility to bypass the postPersist() method for auditing any entity, for may be specific tables only ? 

like - LogRevisionsListener 

Any help will be appreciated. 

Thanks!

**Comment by tolry** [comment]
Hi @ravindrakhokharia,

could you elaborate a bit more on what you are trying to achieve? It is already possible to audit only certain tables/entity classes, see README

##### app/config/config.yml
```yml
simple_things_entity_audit:
    audited_entities:
        - MyBundle\Entity\MyEntity
        - MyBundle\Entity\MyEntity2
```

Or do you need to exclude certain entries of one table from being audited?

**Comment by ravindrakhokharia** [comment]
Hi @tolry 

Sorry for the less information ! 

I do already have 4 entities auditing details, But i found a lots of difference in terms of performance in one of our action. 
We do have two entities in which it flushes at least 600 records at a time in each of the entity, So it adds around 2400 records including Audited entities, which hits the performance issue. Below is the example in one of the environment i did checked with 150 records

for 150 records - when both entities where audited, 
Time it took to response back was 18 secs
While when the audit was removed, 
Time it took to response back was 3 secs

So what i am thinking to do is, bypass the "INS" INSERTION (POSTPERSIST event) audit entries which will decrease the load to add data on very first time. 

So basically want to bypass the POSTPersist event from adding only INSERT entries in to the audit table ! 

Let me know still you need more details. 

Thank you for your reply.

**Comment by dragonito** [comment]
Hi @ravindrakhokharia,

i try to understood your problem.

What you mean with entity saving 150 records?

1. Do you mean the db table and it your script saves 150 rows in one time
2. or do you mean the entity got 150 fields and saving one entity to db will do it very slow?

**Comment by ravindrakhokharia** [comment]
Hi @dragonito 

its 1st option, 
Script saves 150 rows in one time, while the number of columns in the single row is around 9. 
the timing given above was for only 150 rows inserted in the table at a time. 

Let me know if any other details missing. 

Thanks !

**Comment by ravindrakhokharia** [comment]
![modelvariable](https://cloud.githubusercontent.com/assets/2654612/16355539/f339a750-3ad6-11e6-98a9-a3a5f096f911.png)

Here is the example of the simple queries taking too much time in different transactions of the audit tables.

**Comment by ravindrakhokharia** [comment]
@dragonito Does above information makes more clear now ?

@tolry Any news yet ? do you need any information still ? 

Looking forward for the help

**Comment by dragonito** [comment]
@ravindrakhokharia Do you really want to bypass entityAudit for special tables? The result would be inconsistent data. Perhaps it is better not to use entityAudit for those tables.

**Comment by ravindrakhokharia** [comment]
@dragonito But any reason it is taking this much time to run those update queries ? 
Will it be because of the size of the table ?

**Comment by DavidBadura** [comment]
@ravindrakhokharia Would this fix your problem? https://github.com/simplethings/EntityAudit/pull/180

**Comment by ravindrakhokharia** [comment]
@DavidBadura It looks like a relevant solution, i will test the commit on my system, and try to check the performance ! Thanks !

**Comment by ravindrakhokharia** [comment]
@DavidBadura I found some errors while saving data, but I can say it has huge success in terms of performance, 
Some errors like below
An exception occurred while executing 'UPDATE model_audit SET `description` = ? , `is_global` = ? , `is_active` = ? , `created_at` = ? , `updated_at` = ? , `profiler_settings_updated_by_user` = ? , `version_no` = ? , `is_updating_version` = ? , `need_dynamic_weight` = ? , `is_weight_setting_locked` = ? , `parent_id` = ? , `stop_future_use` = ? , `profiler_model_name` = ? , `client_id` = ? , `sorting_option_id` = ? WHERE rev = ? AND id = ?' with params ["Test for model perfomance", false, false, {"date":"2016-09-02 11:25:13.000000","timezone_type":3,"timezone":"UTC"}, {"date":"2016-09-02 11:25:13.000000","timezone_type":3,"timezone":"UTC"}, false, 1, false, false, false, 1184, false, "Test for model perfomance", 14, 1, "15797", 1265]:

Catchable Fatal Error: Object of class DateTime could not be converted to string

Also some values where swapped between the fields

**Comment by DavidBadura** [comment]
solved by #253
